README.md

```diff
@@ -109,14 +88,20 @@ Currently, the following Technical Advisory Groups are active:
 * [TAG Observability](https://github.com/cncf/tag-observability)
 * [TAG Environmental Sustainability](https://github.com/cncf/tag-env-sustainability)
 
+The current Technical Advisory Groups would be spun down during 2025 and replace with new Technical Advisory Groups and TOC Sub Projects, listed below.
+
+## New Technical Advisory Groups and TOC Sub Projects
 
-## Working Groups
+### New Technical Advisory Groups
 
-The TOC has created the following working groups to investigate and discuss the following topics:
+* [TAG Developer Experiance](./tags/tag-developer-experience/)
+* [TAG Infrastructure](./tags/tag-infrastructure/)
+* [TAG Operational REsilience](./tags/tag-operational-resilience/)
+* [TAG Security and Compliance](./tags/tag-security-and-compliance/)
+* [TG Workloads-foundation](./tags/tag-workloads-foundation/charter.md)
 
-| Working Group | Chair            | Meeting Time                          | Minutes/Recordings |
-|---------------|------------------|---------------------------------------|--------------------|
-| [CI](https://github.com/cncf/wg-ci)         |   | Inactive | [Youtube](https://www.youtube.com/playlist?list=PLj6h78yzYM2P3_A3ujWHSxOu1IO_bd7Zi)
-| [Serverless](https://github.com/cncf/wg-serverless) | Ken Owens        |  [Thu of every week at 9AM PT](https://zoom.us/my/cncfserverlesswg)                                     | [Youtube](https://www.youtube.com/playlist?list=PLj6h78yzYM2Ph7YoBIgsZNW_RGJvNlFOt)
+### TOC Sub Projects
 
-All meetings are on the public CNCF calendar: https://goo.gl/eyutah
+* [Contributor Strategy Subproject and Advocacy](./toc_subprojects/contributor-strategy-subproject-and-advocacy/)
```

Comment:
nit: "SubProject" or "Sub Project" for consistency - same for Mentoring and Project Reviews

Reply:
+1 to "SubProject". This makes consistency easier when appearing in file paths, e.g., `toc_subprojectas` vs. `toc_sub_projects`.

Reply:
SubProject +1

Reply:
Updated to SubProject